helm-charts/charts/kube-starrocks/templates/starrocks/starrockscluster.yaml

```diff
@@ -177,23 +197,23 @@ spec:
     nodeSelector:
 {{ toYaml .Values.starrocksBeSpec.nodeSelector | indent 6 }}
 {{- end }}
-{{- if or .Values.starrocksBeSpec.beEnvVars .Values.initPassword.enabled }}
     beEnvVars:
-    {{- if .Values.timeZone }}
       - name: TZ
         value: {{ .Values.timeZone }}
-    {{- end }}
-    {{- if and .Values.initPassword.enabled (.Values.starrocksBeSpec.beEnvVars | toJson | contains "MYSQL_PWD" | not) }}
+      {{- if .Values.datadog.log.enabled }}
```

Comment:
is it possible that there is no env var to be rended at last, and the crd will be invalid. like
```
beEnvVars:
other configure: xxx

Reply:
now, beEnvVars will always has `TZ`.

Reply:
If timeZone is not set , or timeZone=="" in values.yaml, I think it equals set TZ environemnt to "" in container, and also that is the same as not set TZ environment.

pods are all ready:
```
 k get pods
NAME                                      READY   STATUS    RESTARTS   AGE
kube-starrocks-be-0                       1/1     Running   0          98s
kube-starrocks-fe-0                       1/1     Running   0          3m3s
kube-starrocks-operator-b758f6d47-mh2r5   1/1     Running   0          5m27s
```

in pod, like this:
```
env:
    - name: TZ
    - name: POD_NAME
      valueFrom:
        fieldRef:
          fieldPath: metadata.name
```
in container, like this:
```
root@kube-starrocks-fe-0:/opt/starrocks# env | grep TZ
TZ=
root@kube-starrocks-fe-0:/opt/starrocks# date
Mon Jul 31 09:51:07 UTC 2023.   # UTC time
```